install/kubernetes/cilium/templates/clustermesh-config/_helpers.tpl

```diff
@@ -1,14 +1,15 @@
 {{- define "clustermesh-config-generate-etcd-cfg" }}
 {{- $cluster := index . 0 -}}
 {{- $domain := index . 1 -}}
+{{- $prefix := ternary "common-" (printf "%s." $cluster.name) (or (empty ($cluster.tls).cert) (empty ($cluster.tls).key)) -}}
```

Comment:
do we need theses parenthesis?
```suggestion
{{- $prefix := ternary "common-" (printf "%s." $cluster.name) (or (empty $cluster.tls.cert) (empty $cluster.tls.key)) -}}
```

Reply:
They are a workaround to prevent issues in case `tls` is nil (which could happen, since by default all that section is commented out in the values file. Ref: https://stackoverflow.com/a/68807258

Reply:
@giorio94 :exploding_head: TIL about this thank you! Super useful to have and at the same time really not the most intuitive syntax IMO.

Reply:
I completely agree that this is not intuitive at all, and I also struggled quite a lot to get a working solution. I'll add a comment to explain why parenthesis are required in this case.